### PR TITLE
Feat: Add powerman role

### PR DESCRIPTION
**Pull request by oxedions**
Add task, handler and template.
Missing readme.

### CODE REVIEW

**Comment by actatux** [comment]
To solve the Test RPM build check you need to add `powerman` to https://github.com/bluebanquise/bluebanquise/blob/ca41671e349d61bb54783898ad7187d2c91af33d/bluebanquise.spec#L3-5.

**Comment by oxedions** [comment]
Thanks a lot !

**Comment by oxedions** [comment]
Ready to be reviewed.

File: roles/addons/powerman/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      - freeipmi
+      - powerman
+    state: present
+  tags:
+    - package
+
+- name: template █ Generate /etc/powerman/powerman.conf
+  template:
+    src: powerman.conf.j2
+    dest: powerman/powerman.conf
```
Comment (actatux) [inline roles/addons/powerman/tasks/main.yml:14]: typo?
```suggestion
    dest: /etc/powerman/powerman.conf
```

Comment (oxedions) [reply roles/addons/powerman/tasks/main.yml:14]: Good catch !

**Comment by actatux** [comment]
I did not notice yesterday, but the static code analysis test is failing.

**Comment by oxedions** [comment]
Yes, I have seen this failing static check. I skipped it as non related to the PR in progress :-)

**Pull request merged**